test/test_api.py

```diff
@@ -14,126 +12,111 @@
 
 current_path = Path(os.path.dirname(os.path.realpath(__file__)))
 
+geoflow_data_path = current_path / "meshfiles" / "ugrid" / "geoflow-small"
+gridfile_geoflow = current_path / "meshfiles" / "ugrid" / "geoflow-small" / "grid.nc"
+geoflow_data_v1 = geoflow_data_path / "v1.nc"
+geoflow_data_v2 = geoflow_data_path / "v2.nc"
+geoflow_data_v3 = geoflow_data_path / "v3.nc"
 
-class TestAPI(TestCase):
-    geoflow_data_path = current_path / "meshfiles" / "ugrid" / "geoflow-small"
-    gridfile_geoflow = current_path / "meshfiles" / "ugrid" / "geoflow-small" / "grid.nc"
-    geoflow_data_v1 = geoflow_data_path / "v1.nc"
-    geoflow_data_v2 = geoflow_data_path / "v2.nc"
-    geoflow_data_v3 = geoflow_data_path / "v3.nc"
-
-    gridfile_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
-    dsfile_var2_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30_var2.nc"
-    dsfiles_mf_ne30 = str(
-        current_path) + "/meshfiles/ugrid/outCSne30/outCSne30_*.nc"
-
-    def test_open_geoflow_dataset(self):
-        """Loads a single dataset with its grid topology file using uxarray's
-        open_dataset call."""
+gridfile_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
+dsfile_var2_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30_var2.nc"
+dsfiles_mf_ne30 = str(current_path) + "/meshfiles/ugrid/outCSne30/outCSne30_*.nc"
 
-        # Paths to Data Variable files
-        data_paths = [
-            self.geoflow_data_v1, self.geoflow_data_v2, self.geoflow_data_v3
-        ]
+def test_open_geoflow_dataset():
+    """Loads a single dataset with its grid topology file using uxarray's
+    open_dataset call."""
 
-        uxds_v1 = ux.open_dataset(self.gridfile_geoflow, data_paths[0])
+    # Paths to Data Variable files
+    data_paths = [
```

Comment:
`test_copy_dataarray` below also defines such a `data_paths` var, but I am not sure either of these test cases actually needs to define it (since they only access one data file out of them). If still yes, then for reusing the code, maybe define it in the beginning of the test script instead, and then just access whatever needed from each test case, or don't define it at all and just access the actual data file needed